torch/_lazy/visit.py

```diff
@@ -0,0 +1,67 @@
+import torch
+
+
+def visit_structure(structure, select_fn, apply_fn, scope=None, strict=True):
+    """Recursively traverse nested structureure and calls the apply_fn callable on
```

Comment:
hmm there is this 'pytree flatten' infra that is available. i am wondering if it is directly usable for you or not.

https://github.com/pytorch/pytorch/blob/b8b54eccd293032ed6761ac21c72b1adf27999d3/torch/utils/_pytree.py

I think if you just want to hit all the tensors with your 'apply_fn' you can directly use pytree and not have to write all this code.

Reply:
Thanks for the suggestion. I wasn't aware of this util